src/meta_schedule/schedule_rule/multi_level_tiling_with_intrin.cc

```diff
@@ -55,8 +55,10 @@ class MultiLevelTilingWithIntrinNode : public MultiLevelTilingNode {
     auto res = MultiLevelTilingNode::Apply(sch->Copy(), block_rv);
 
     if (res.empty()) {
+      TVM_PY_LOG(INFO, logging_func) << "The workload cannot be tensorized.";
```

Comment:
What is the difference between hitting this and `!CheckAutoTensorizeApplicable`?

Reply:
also added logging in the above case, `CheckAutoTensorizeApplicable` is non-exhaustive, tensorization can still fail even if it passes the check